### ISSUE TITLE
netlify-cms doesn't pass @netlify/plugin-a11y test

### ISSUE THREAD

**Issue by jsjoeio** [netlify/netlify-cms#6061]
**Describe the bug**
The "Login with Netlify" button does not have sufficient contrast.

```logs
Error: Elements must have 
sufficient
 color contrast (https://dequeuniversity.com/rules/axe/4.3/color-contrast?application=axeAPI)
10:53:25 PM:  ├── color-contrast
10:53:25 PM:  ├── #nc-root > div > section > button
10:53:25 PM:  └── <button class="css-tu1it6-LoginButton-button-default-gray-disabled e4hp3ji4">Login with Netlify Identity</button>
```

**To Reproduce**
1. Add Netlify CMS to any site
2. Add ` package = "@netlify/plugin-a11y"` to `nelify.toml`
3. Deploy site to Netlify

**Expected behavior**
`netlify-cms` should pass `@netlify/plugin-a11y` test 😅 

**Screenshots**
![image](https://user-images.githubusercontent.com/3806031/146631191-d5be7ed3-0149-4b98-9c06-9c5965567f05.png)

**Applicable Versions:**

 - Netlify CMS version: 2.0.0
 - Git provider: GitHub
 - OS: Windows 10
 - Browser version [e.g. chrome 22, safari 11]

 - Node.JS version:

**CMS configuration**
N/A

**Additional context**
Possible duplicate of https://github.com/netlify/netlify-cms/issues/1333

**Comment by erezrokah** [comment]
Thanks for putting this together @jsjoeio!

Do you mind adding this to #1333 so we can have a centralized place to handle accessibility related issues?

The CMS is a community project, so it would be great to see some contributions around this.
I labeled this issue and #1333 and `good first issue` to help with that.

**Comment by jsjoeio** [comment]
> Do you mind adding this to #1333 so we can have a centralized place to handle accessibility related issues?

Sure, I'll do that now!

> The CMS is a community project

Ah, for some reason, I assumed since it's under the `netlify` org/namespace that they maintained it. Good to know!

**Comment by jsjoeio** [comment]
Closing in favor of https://github.com/netlify/netlify-cms/issues/1333#issuecomment-998115794

**Comment by erezrokah** [comment]
> Ah, for some reason, I assumed since it's under the `netlify` org/namespace that they maintained it. Good to know!

We (Netlify) have dedicated resources for it (I'm one of those resources, 👋) which are targeted to support contributions.
I hope that helps clarifying my initial comment.

**Comment by jsjoeio** [comment]
> We (Netlify) have dedicated resources for it (I'm one of those resources, 👋) which are targeted to support contributions.
I hope that helps clarifying my initial comment.

Ah, I misunderstood. I thought you were a person from the community volunteering to run the project in your free time on behalf of Netlify 😅  Thanks for clarifying!

Another question came to mind. How do we (the community) know what issues you (the maintainers) are prioritizing? I peeked in the milestones and projects but don't see those being used. Just curious in case I run into other issues.

**Comment by erezrokah** [comment]
Hi @jsjoeio, any active work is done by external contributors, either by picking good first issues, or by looking at most [upvoted issues](https://github.com/netlify/netlify-cms/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc).

As maintainers we focus on reviewing PRs that come in and guiding people in resolving issues.
